src/main/java/frc/robot/Constants.java

```diff
@@ -19,11 +24,87 @@ public class Constants {
+
+    public static final Pose2d RED_LEFT_FAR_SIDE_CORAL_STATION = new Pose2d((17.548225 - 1.6382254362106323), 0.621896505355835, Rotation2d.fromDegrees(126));
+    public static final Pose2d RED_LEFT_NEAR_SIDE_CORAL_STATION = new Pose2d((17.548225 - 0.5229208469390869), 1.349269151687622, Rotation2d.fromDegrees(126));
+    public static final Pose2d RED_RIGHT_NEAR_SIDE_CORAL_STATION = new Pose2d((17.548225 - 0.5559024810791016), 6.728616237640381, Rotation2d.fromDegrees(234));
+    public static final Pose2d RED_RIGHT_FAR_SIDE_CORAL_STATION = new Pose2d((17.548225 - 1.5863468647003174), 7.540848731994629, Rotation2d.fromDegrees(234));
+
+    static final List<Pose2d> blueBranchPoses = List.of(BLUE_BRANCH_A, BLUE_BRANCH_B,
+            BLUE_BRANCH_C, BLUE_BRANCH_D, BLUE_BRANCH_E, BLUE_BRANCH_F,
+            BLUE_BRANCH_G, BLUE_BRANCH_H, BLUE_BRANCH_I, BLUE_BRANCH_J,
+            BLUE_BRANCH_K, BLUE_BRANCH_L);
+
+    static final List<Pose2d> redBranchPoses = List.of(RED_BRANCH_A, RED_BRANCH_B,
+            RED_BRANCH_C, RED_BRANCH_D, RED_BRANCH_E, RED_BRANCH_F,
+            RED_BRANCH_G, RED_BRANCH_H, RED_BRANCH_I, RED_BRANCH_J,
+            RED_BRANCH_K, RED_BRANCH_L);
+
+    static final List<Pose2d> blueCoralStationPoses = List.of(BLUE_LEFT_NEAR_SIDE_CORAL_STATION, BLUE_LEFT_FAR_SIDE_CORAL_STATION,
+            BLUE_RIGHT_NEAR_SIDE_CORAL_STATION, BLUE_RIGHT_FAR_SIDE_CORAL_STATION);
+
+    static final List<Pose2d> redCoralStationPoses = List.of(RED_LEFT_NEAR_SIDE_CORAL_STATION, RED_LEFT_FAR_SIDE_CORAL_STATION,
+            RED_RIGHT_NEAR_SIDE_CORAL_STATION, RED_RIGHT_FAR_SIDE_CORAL_STATION);
+
+    private static Pose2d getBranchPose(int number, boolean adding, double xOffset, double yOffset){
+        if (adding) {
+            return new Pose2d(Units.inchesToMeters(Math.hypot(xBranchDistanceFromCenter, yBranchDistanceFromCenter) *
+                    Math.cos(((Math.PI / 3) * number) + Math.asin(yBranchDistanceFromCenter / xBranchDistanceFromCenter)) + xOffset),
+                    Units.inchesToMeters(Math.hypot(xBranchDistanceFromCenter, yBranchDistanceFromCenter) * Math.sin(((Math.PI / 3) * number) +
+                            Math.asin(yBranchDistanceFromCenter / xBranchDistanceFromCenter)) + yOffset),
+                    Rotation2d.fromRadians((Math.PI / 3) * number));
+        }
+        else {
+            return new Pose2d(Units.inchesToMeters(Math.hypot(xBranchDistanceFromCenter, yBranchDistanceFromCenter) *
+                    Math.cos(((Math.PI / 3) * number) - Math.asin(yBranchDistanceFromCenter / xBranchDistanceFromCenter)) + xOffset),
+                    Units.inchesToMeters(Math.hypot(xBranchDistanceFromCenter, yBranchDistanceFromCenter) * Math.sin((Math.PI / 3) * number -
+                            Math.asin(yBranchDistanceFromCenter / xBranchDistanceFromCenter)) + yOffset),
+                    Rotation2d.fromRadians((Math.PI / 3) * number));
+        }
```

Comment:
Instead of using `Units.inchesToMeters` inside the `getBranchPose` method, your inputs to the `getBranchPose` method should already be in meters.